### ISSUE TITLE
Animation CSS properties not working

### ISSUE THREAD

**Issue by lyqht** [argyleink/open-props#102]
I'm using astro build for my app and trying to include open-props' individual pack of animation into my  project.

I have used `npm install open-props`, and I try to import that pack using
```
@import "open-props/animations";
```

and used the default animation example given on the website

```
.slide-fade {
  animation:
    var(--animation-fade-out) forwards,
    var(--animation-slide-out-down);
  animation-timing-function: var(--ease-squish-3);
  animation-duration: 1s;
}
```

but I'm not able to see my component being animated. In the Inspector, it seems that the CSS properties for `--animation` are undefined.

**Comment by roblevintennis** [comment]
I'm just learning how to use open-props myself so please bare with me, but, I do think there's a discrepancy between the guidelines on the main OP page in terms of just importing it using NPM from an `index.css` page. Specifically:

![image](https://user-images.githubusercontent.com/142403/144752825-1901baac-2c18-4c63-b23a-8e353973fc42.png)


For example, see
https://unpkg.com/browse/open-props@1.0.9/dist/open-props.min.css

vs.

https://unpkg.com/browse/open-props@1.0.9/open-props.min.css

I've set up a branch on my fork that pulls in open-props as an npm package. I have not set up astro or postcss or anything, so I have to import it "manually" from node modules. So, the import paths are different, but you can see that if I directly import the top-level open-props.min.css it works but not the one in dist which doesn't appear to actually be css but a js module of some sorts.

See:

```css
/* DOES NOT WORK WHILE FOLLOWING LINE DOES @import "./node_modules/open-props/dist/open-props.min.css"; */
@import "./node_modules/open-props/open-props.min.css";
```

Here's my branch:

https://github.com/argyleink/open-props/compare/main...roblevintennis:examples-directory?expand=1#diff-e5097947cb80bfaa9400acb49df59797fbf80b73e00e4b4844206e3a19b98395R1

I don't quite understand the intent of this `dist/open-props.min.css` but personally, I would expect that OP generates that to be like the top-level `open-props.min.css` which is in fact importable from CSS, or, rename it to something that indicates its expected to be loaded from JavaScript.

Again, I'm new to the library so probably @argyleink will have something more to add.

**Comment by argyleink** [comment]
Ohh no, sorry this is happening! The npm exports have been troublesome. 

I can check this out today, I'll try and roll out a fix asap after thoroughly testing individual imports from npm.

**Comment by argyleink** [comment]
For now the CDN can unblock you maybe?

**Comment by argyleink** [comment]
also, which version of astro! .19 is very different than .21, as .21 has vite included, a tool very good at resolving package.json export fields.

**Comment by argyleink** [comment]
interesting, this file `open-props/dist/open-props.min.css` shouldnt exist, appears i have an error in my js module build system to debug

**Comment by argyleink** [comment]
> interesting, this file `open-props/dist/open-props.min.css` shouldnt exist, appears i have an error in my js module build system to debug

just pushed a fix for this. shouldnt have been what was making individual imports not work, but def removes confusion and properly named the umd module

**Comment by argyleink** [comment]
Forked the Astro starter and have open props loading just the animations, [check it out here](https://stackblitz.com/edit/github-hqvgfx?file=src%2Fpages%2Findex.astro). I think the issue was the animations used the easings, and they fail if the easings arent loaded? either that or it's the version of Astro you're on struggles with the export field of package.json, so it might be easier to get the individual bundles from the cdn? Let me know how to help more, I think you're unblocked now?!

**Comment by lyqht** [comment]
Thanks for the help @argyleink @roblevintennis 😄

I didn't realize that the easings is also an individual package by itself. On my current version `"open-props": "^1.0.9"` if I did `@import "open-props/easings";`, I'm getting

```
[vite] Internal server error: Failed to find 'open-props/easings'
  in [
    /home/lyqht/Github/advents/advent-of-css/day4-keyboard-jiggle/src/styles
  ]
  Plugin: vite:css
```

[Switching to the CDN link]((https://github.com/lyqht/keyboard-simulator/blob/main/src/styles/keyboard.css)) worked, thank you! 😄

**Comment by argyleink** [comment]
I fixed the easing export in `1.0.11`!
